server/controllers/listing.js

```diff
@@ -16,6 +16,16 @@ class ListingController {
       next(ApiError.internal(`${err}`));
     }
   };
+
+  getAllListings = async (req, res, next) => {
+    try {
+      const listing = await this.listingService.getAllListings(req.params);
+   
```

Comment:
Super super super picky: just point this out, you don't need to remove this extra empty line lol

Reply:
All good